web/src/components/datasets/DatasetTags.tsx

```diff
@@ -112,22 +168,81 @@ const DatasetTags: React.FC<IProps> = (props) => {
 
   return (
     <>
+      <Snackbar
+        open={snackbarOpen}
+        autoHideDuration={500}
+        style={{ zIndex: 9999 }}
+        onClose={() => setSnackbarOpen(false)}
+        message={'Tag updated.'}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+      />
       <Box display={'flex'} alignItems={'center'}>
         <MQText subheading>{i18next.t('dataset_tags.tags')}</MQText>
         {formatTags(datasetTags, tagData)}
-        <MQTooltip placement='top' title={i18next.t('dataset_tags.tooltip')} key='tag-tooltip'>
-          <IconButton
-            onClick={openDialog}
+        <ButtonGroup
+          variant='contained'
+          ref={anchorRef}
+          aria-label='Tags Nested Menu'
+          sx={{ height: '30px', width: '20px', marginLeft: '5px' }}
```

Comment:
Try to use units of four for these kinds of operations. I believe if you say `1` or `2` that does the multiplication for you.